docs/visualize.asciidoc

```diff
@@ -1,2 +1,131 @@
 [[visualize]]
-== Visualizing your Data
\ No newline at end of file
+== Visualize
+
+You can use Kibana's _Visualize_ app to design and create data visualizations. You can save these visualizations, use
+them individually, or combine visualizations into a _dashboard_. A visualization can be based on one of the following 
```

Comment:
My personal style would be to shorten the intro to, "A visualization can be based on:"